recipes/unyt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "unyt" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "4a135bdcd79fcb4130f16ca8665ffc3407e2af6f83f2e7debcc2a7459c4e0f01" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # we can enable this when we drop py27 or when conda-forge
+  # lets you do this with optional py27 dependencies
+  # noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.13.0
```

Comment:
setup.py says `numpy >1.13.0`. Which one is correct?

Reply:
The setup.py is incorrect. I can cut another release to fix that if it matters.